.gitignore

```diff
@@ -55,3 +55,7 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# Misc
```

Comment:
As you can we have two different uses, it is better to leave each developer add its own ignores outside.